diagnostic.md

```diff
@@ -149,9 +173,11 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
+Divs 2 and 3 would be on the first row, aliged left to right in this order.
 
-<!-- your answer ends here -->
+Divs 4 and 5 would be on the second row, aligned left to right in this order.
+
+Div 1 would float to the right side of the page. It's not really aligned to any of the two rows because it has been taken out of the document flow.
```

Comment:
div 1 is on the top row with divs 2 & 3.

.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5